superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -66,7 +69,22 @@ def _init_distributed_setting(self):
                     )
                     return False
 
-                torch.distributed.init_process_group(backend=self._args.distributed_backend.value)
+                if self.version_larger(str(torch.__version__), '1.9.0'):
```

Comment:
```py
from packaging import version
if version.parse(torch.__version__) >= version.parse('1.9.0'):
   ...
```

Reply:
seems you need to compare with '1.9.0a0' instead of '1.9.0'?

or could you just test the api changes instead of comparing version? for example (not verified)
```py
try:
    from torch.distributed import elastic
    # >= 1.9.0a0
except ImportError:
    # < 1.9.0a0
```

Reply:
really thanks, it works, updated to use api to check